### PR TITLE
Properly clean up 'LLVMBuildFactory.enable_projects' attr from the runtime projects.

### PR DESCRIPTION
**Pull request by vvereschaka**
Remove all possible runtime projects from 'depends_on_project' argument when
preparing a list of enabled projects within the LLVMBuildFactory factory (enable_projects).

### CODE REVIEW

**Review by slydiman — APPROVED** [review]
LGTM

File: zorg/buildbot/process/factory.py [hunk 1]
```diff
@@ -175,5 +175,5 @@ def addGetSourcecodeForProject(self, project, name=None, src_dir=None, **kwargs)
                 progress=True,
                 workdir=util.Interpolate(src_dir),
                 retryFetch=True,
-                clobberOnFailure=True,                
+                clobberOnFailure=True,
```
Comment (gkistanova) [inline zorg/buildbot/process/factory.py:178]: Please remove unrelated cosmetic changes from this patch.

**Review by gkistanova — CHANGES_REQUESTED** [review]
Some of the runtimes could be built as projects. And this is Ok and supported. This patch breaks this.
